pkg/core/interop/names/names.go

```diff
@@ -0,0 +1,66 @@
+package names
+
+var names = []string{
```

Comment:
This is what I wanted to avoid --- yet another name registry duplicating declarations made by interops.

Reply:
This is a perfect case for some code-generation tool, but Go isn't very friendly to those, so as a compromise I think we can make the following:
 * add `interopnames` (not `names`, please) package that would just contain string constants for all interop functions.
 * use its constants for interop registrations
 * use its constants for id-to-name in vm/cli

At least that would remove string duplication that is very easy to make error in. It still leaves some potential for constant usage to get out of sync, but that's unavoidable given the constraints we have.